pkg/localmetrics/localmetrics.go

```diff
@@ -88,9 +88,28 @@ func UpdateAWSMetrics(totalAccounts int) {
 	MetricTotalAWSAccounts.With(prometheus.Labels{"name": "aws-account-operator"}).Set(float64(totalAccounts))
 }
 
+// UpdateAccountCRUnclaimedMetric updates the unclaimed account metric
+func UpdateAccountCRUnclaimedMetric(accountList awsv1alpha1.AccountList) {
+
+	unclaimedAccountCount := 0
+
+	for _, account := range accountList.Items {
+		if account.Status.Claimed == false && account.Status.Reused == false {
+			if account.Status.State == "Ready" || account.Status.State == "PendingVerifiation" {
+				unclaimedAccountCount++
+			}
+		}
+	}
+
+	MetricTotalAccountCRsUnclaimed.With(prometheus.Labels{"name": "aws-account-operator"}).Set(float64(unclaimedAccountCount))
+
+}
+
 // UpdateAccountCRMetrics updates all metrics related to Account CRs
 func UpdateAccountCRMetrics(accountList *awsv1alpha1.AccountList) {
-	unclaimedAccountCount := 0
+
+	go UpdateAccountCRUnclaimedMetric(*accountList)
```

Comment:
Since this is a go routine, do we need to think on using any kind of sync/WaitGroup approach?

Reply:
I believe we'll need to sync because is it possible these could run simultaneously? We would get an API error if it did

Reply:
Since this is just taking a copy of a struct, there's no danger in leaving it as is

Reply:
a waitgroup might be nice to limit the number of concurrent threads

Reply:
Is it possible to have multiple calls of that go routine running at the same time? What if we call `MetricTotalAccountCRsUnclaimed.With(prometheus.Labels{"name": "aws-account-operator"}).Set(float64(unclaimedAccountCount))` simultaneously ?

Reply:
Just a thought here, if we are not waiting for the the go routine to finish what is the benefit of having it in a go routine? @rogbas @fahlmant

Reply:
We are waiting for it to return.